packages/amplify-graphql-api-construct/src/internal/asset-provider.ts

```diff
@@ -0,0 +1,64 @@
+import * as fs from 'fs';
```

Comment:
This file was moved from `packages/amplify-graphql-api-construct/src/internal/asset-manager.ts`. There are enough line changes that git is representing this as removing one file and adding another.